eventtracking/django/django_tracker.py

```diff
@@ -0,0 +1,175 @@
+"""A django specific tracker"""
+
+from __future__ import absolute_import
```

Comment:
Just moved the code from `eventtracking/django/__init__.py` file to this new file. This is to avoid the `Models aren't ready yet` errors we get while importing model at top level.

This doesn't affect the functionality of eventtracking but it does cause the tests in edx-platform to fail.
in edx-platform the following two files import `DjangoTracker` as `from eventtracking.django import DjangoTracker`:

```
common/djangoapps/track/tests/__init__.py
common/djangoapps/track/tests/test_segment.py
```

Changing this import to `from eventtracking.django.django_tracker import DjangoTracker` will fix those tests. This means we will have to change edx-platform once we have published a new release of this app.

Alternate to this would be that we have to import all model imports inside the methods.